src/__stories__/grid-story.tsx

```diff
@@ -0,0 +1,262 @@
+                            <SnapCard
+                                icon={
+                                    <Circle size={40} backgroundColor={skinVars.colors.brandLow}>
+                                        <IconAcademicRegular color={skinVars.colors.brand} />
+                                    </Circle>
+                                }
+                                title="Title 1"
+                                subtitle="Subtitle 1"
+                            />
+                        </GridItem>
+                        <GridItem rowSpan={3}>
+                            <SnapCard
+                                icon={
+                                    <Circle size={40} backgroundColor={skinVars.colors.brandLow}>
+                                        <IconAcademicRegular color={skinVars.colors.brand} />
+                                    </Circle>
+                                }
+                                title="Title 2"
+                                subtitle="Subtitle 2"
+                            />
+                        </GridItem>
+                        <GridItem rowSpan={2} columnSpan={2}>
+                            <SnapCard
+                                icon={
+                                    <Circle size={40} backgroundColor={skinVars.colors.brandLow}>
+                                        <IconAcademicRegular color={skinVars.colors.brand} />
+                                    </Circle>
+                                }
+                                title="Title 3"
+                                subtitle="Subtitle 3"
+                            />
+                        </GridItem>
+                    </Grid>
+                </Stack>
+            </Stack>
+        </Box>
+    </ResponsiveLayout>
+);
+
+Default.storyName = 'Grid';
+
+Default.args = {
+    gap: 16,
+};
+
+type AutoColumnsArgs = {
+    gap: React.ComponentProps<typeof Grid>['gap'];
+    columnsMinSize: number;
+    numItems: number;
+};
+
+export const AutoColumns: StoryComponent<AutoColumnsArgs> = ({gap, columnsMinSize, numItems}) => (
+    <ResponsiveLayout>
+        <Box paddingY={24}>
+            <Grid columns={{minSize: columnsMinSize}} gap={gap}>
+                {Array.from({length: numItems}, (_, idx) => (
+                    <SnapCard
+                        key={idx}
+                        icon={
+                            <Circle size={40} backgroundColor={skinVars.colors.brandLow}>
+                                <IconAcademicRegular color={skinVars.colors.brand} />
+                            </Circle>
+                        }
+                        title={`Title ${idx + 1}`}
+                        subtitle="Subtitle"
+                    />
+                ))}
+            </Grid>
+        </Box>
+    </ResponsiveLayout>
+);
+
+AutoColumns.storyName = 'Auto columns';
+AutoColumns.args = {
+    gap: 16,
+    columnsMinSize: 100,
+    numItems: 10,
```

Comment:
Maybe we should also allow setting width/height in controls?

Reply:
I don't feel it would be very useful in this story